scripts/functions

```diff
@@ -1305,6 +1305,24 @@ CT_DoBuildTargetTuple() {
     CT_ARCH_TARGET_LDFLAGS="${CT_ARCH_TARGET_LDFLAGS} ${CT_ARCH_ENDIAN_LDFLAG}"
 }
 
+# This function determines the target tuple for a given set of compiler
+# flags, using either GCC's multiarch feature (if supported) or using
+# fallback architecture-specific functions.
+CT_DoMultilibTarget() {
+    local multi_flags="$1"
+    local target="$2"
+    local gcc_multiarch
+
+    gcc_multiarch=$("${CT_TARGET}-gcc" -print-multiarch ${multi_flags})
```

Comment:
Oh, and redirect `stderr` to `/dev/null` for good measure (since the
comment seems to imply that gcc may choke on this).